.run/db down -v.run.xml

```diff
@@ -1,6 +1,6 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="db down -v" type="ShConfigurationType">
-    <option name="SCRIPT_TEXT" value="docker compose down -v db" />
+    <option name="SCRIPT_TEXT" value="docker compose down -v" />
```

Comment:
New version of `docker compose down` doesn't let you specify a container for some reason anymore, at least it did for me. I think the name is still ok, since it does turn it down the db, I am open to having new name suggestions

Reply:
just tried it and it seems like it works again, so I put it back to the way it was. thanks for flagging it.